bootstrap/applicationset/applicationset-bootstrap.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: argoproj.io/v1alpha1
+kind: ApplicationSet
+metadata:
+  name: bootstrap
+  namespace: openshift-gitops
+spec:
+  generators:
+  - list:
+      elements:
+      - cluster: in-cluster
+        url: https://kubernetes.default.svc
+        name: ic-shared-minio-app
+        repoURL: https://github.com/rh-aiservices-bu/insurance-claim-processing.git
+        targetRevision: dev
+        path: bootstrap/ic-shared-minio
+      - cluster: in-cluster
+        url: https://kubernetes.default.svc
+        name: ic-shared-database-app
+        repoURL: https://github.com/rh-aiservices-bu/insurance-claim-processing.git
+        targetRevision: dev
+        path: bootstrap/ic-shared-database
+      - cluster: in-cluster
+        url: https://kubernetes.default.svc
```

Comment:
defined multiple times, but not used. (hard-coded in the bottom part). 
Is there a reason to keep these lines here in the top part? @guimou